engine/src/build/toolchain/apple/get_tool_mtime.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Does this need a shebang?

Reply:
Not officially but it makes it a heck of a lot easier to debug locally. Done.